homeassistant/components/homekit_controller/humidifier.py

```diff
@@ -248,6 +249,92 @@ def unique_id(self) -> str:
         return f"homekit-{serial}-{self._iid}-{self.device_class}"
 
 
+class HomeKitDiffuser(HomeKitEntity, HumidifierEntity):
```

Comment:
Reluctant to call it HomeKitDiffuser, as it's not an official Apple service and its specific to devices that have the vocolinc spray level characteristic. I think i'd prefer to call it VocolincDiffuser or even VocolincFlowerbud.